.travis.yml

```diff
@@ -13,4 +13,4 @@ install:
   - php composer.phar install
 
 script:
-  - php vendor/bin/phpunit -c tests/phpunit.xml
\ No newline at end of file
+  - composer test
```

Comment:
Keeping the travis yml clean and simple, I like it :)

Reply:
:+1: